backend/de.metas.adempiere.adempiere/base/src/main/java-legacy/org/compiere/util/SQLValueStringResult.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * #%L
+ * de.metas.adempiere.adempiere.base
+ * %%
+ * Copyright (C) 2022 metas GmbH
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 2 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program. If not, see
+ * <http://www.gnu.org/licenses/gpl-2.0.html>.
+ * #L%
+ */
+
+package org.compiere.util;
+
+import lombok.Builder;
+import lombok.Value;
+
+import javax.annotation.Nullable;
+import java.sql.SQLWarning;
+
+@Value
+@Builder
+public class SQLValueStringResult
+{
+	@Nullable
+	String returnedValue;
+
+	@Nullable
+	SQLWarning warning;
```

Comment:
about "polluting" this class with a java.sql.* dependency..suggestion: better iterate the warning/nextwarnings in DB.java (or a util-class) and have a list in the SQLValueStringResult ?